code/pizzabot.js

```diff
@@ -1,7 +1,37 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian"
+const hawaiian = "Hawaiian"
+const pepperoni = "Pepperoni"
+const pizzaPrice = 80;
 
-const pizzaPrice = 80
+alert(`Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`)
+let orderName = prompt('Enter the name of the pizza you want to order today.');
 
-//Put your Javscript code here:
+const validateOrderName = () => { 
+    if (orderName.toLowerCase() === vegetarian.toLowerCase() || orderName.toLowerCase() === hawaiian.toLowerCase() || orderName.toLowerCase() === pepperoni.toLowerCase()) { 
+        orderQuantity = prompt(`How many ${orderName} pizzas do you want?`);
+        return true; 
+    } else {
+        alert("Select a pizza from the menu")
+        prompt('Enter the name of the pizza you want to order today.');
+        validateOrderName();
```

Comment:
Clever :) FIY, this approach is called `recursive function` - it means that function is called by itself during execution